lightkurve/tests/test_nexsci.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import pytest
+import warnings
+
+from ..nexsci import *
+from .. import nexsci
+
+
+def test_retrieval():
+    ''' Test the data can be retrieved '''
+    nexsci._retrieve_online_data()
+
+def test_freshness():
+    ''' Test the fresh data works '''
+    # This function should always run...
+    nexsci._check_data_is_fresh()
+
+def test_dataframe():
+    ''' Test the dataframe can be read '''
+    df = get_nexsci_data()
+    assert(len(df) >= 0)
+
+def test_trappist():
+    ''' Test the planet mask works '''
+    t = np.arange(0, 100, 0.001)
+    m = find_planet_mask('TRAPPIST-1', t)
+    # Trappist 1 should have a planet transiting 7.9% of the time
+    assert np.isclose((1 - (m.sum()/len(t))), 0.079, atol=0.001)
```

Comment:
This assertion fails in python 2.7 because `sum()` is returning an int. I believe a quick fix is to replace `m.sum()/len(t)` with `float(m.sum())/len(t)`.